src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SignedXml.cs

```diff
@@ -1115,6 +1115,11 @@ private bool CheckSignedInfo(KeyedHashAlgorithm macAlg)
 
         private static bool IsKeyTheCorrectAlgorithm(AsymmetricAlgorithm key, Type expectedType)
         {
+            if (expectedType is null)
```

Comment:
Please share a repro for how this would actually happen in practice.

Reply:
I have no repro. This is a warning of static code analyzer.

Reply:
Which can be false positives. Hence asking for a repro.

Reply:
I do not insist on the fact that current implementation is inaccurate. After discussing it with a colleague we decided to try to make this change.
I'll talk to him one more time.
So, is my understanding correct that you find this change unnecessary?

Reply:
I agree that this change is unnecessary in present.
I'll close the PR.
Thanks for your participation!